docs/wustl_htcf.md

```diff
@@ -0,0 +1,21 @@
+# nf-core/configs: WUSTL High Throughput Computing Facility Configuration
+
+Forked from https://github.com/nf-core/configs/blob/master/docs/prince.md
+
+## nf-core pipelines that use this repo
```

Comment:
I would all all the information about the `singularity` cqache an the $LAB requirement here too, not just as warnings in teh config itself